cloud/deploy/deploy.go

```diff
@@ -103,7 +103,7 @@ func deployDags(path string, deployInfo *deploymentInfo, client astro.Client) er
 		status = "FAILED"
 	}
 
-	_, err = deployment.ReportDagDeploymentStatus(dagDeployment.ID, deployInfo.deploymentID, action, versionID, status, message, client)
+	_, err = deployment.ReportDagDeploymentStatus(dagDeployment.ID, runtimeID, action, versionID, status, message, client)
```

Comment:
Does the `ReportDagDeploymentStatus()` need to handle `deploymentID` as input ever? What if dags were deployed using a `deploymentID` instead of `runtimeID`?

Do we need unit test cases for that scenario?